packages/api-server/api_server/routes/tasks/scheduled_tasks.py

```diff
@@ -0,0 +1,115 @@
+import asyncio
+from datetime import datetime
+
+import schedule
+import tortoise.transactions
+from fastapi import Depends, HTTPException
+from pydantic import BaseModel
+
+from api_server.authenticator import user_dep
+from api_server.dependencies import pagination_query
+from api_server.fast_io import FastIORouter
+from api_server.models import DispatchTaskRequest, Pagination, TaskRequest, User
+from api_server.models import tortoise_models as ttm
+from api_server.repositories import TaskRepository, task_repo_dep
+
+from .tasks import post_dispatch_task
+
+router = FastIORouter(tags=["Tasks"])
+
+
+class PostScheduledTaskRequest(BaseModel):
+    task_request: TaskRequest
+    schedules: list[
+        ttm.ScheduledTaskSchedulePydantic
+    ]  # a scheduled task can have multiple schedules
+
+
+async def schedule_task(task: ttm.ScheduledTask, task_repo: TaskRepository):
+    await task.fetch_related("schedules")
+    jobs = [x.to_job() for x in task.schedules]
```

Comment:
Maybe put this in a try except block since `to_job` raises an exception in the case of an unsupported period?

Reply:
`kp/scheduling-frontend` actually addressed this.